src/main/java/ru/javarush/quest/entity/Account.java

```diff
@@ -0,0 +1,64 @@
+package ru.javarush.quest.entity;
+
+public class Account {
+    private Long id;
+    private final String name;
+    private final String login;
+    private String password;
+
+    public Account(Long id, String name, String login, String password) {
+        this.id = id;
+        this.name = name;
+        this.login = login;
+        this.password = password;
+    }
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getLogin() {
+        return login;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (!(o instanceof Account)) return false;
+
+        Account account = (Account) o;
+
+        if (!id.equals(account.id)) return false;
+        if (!name.equals(account.name)) return false;
+        if (!login.equals(account.login)) return false;
+        return password.equals(account.password);
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
let's use `Objects.hash` for simplification

Reply:
i deleted this hash method.